lib/v8-log-to-ticks.js

```diff
@@ -69,7 +69,7 @@ function fixLines (line) {
 
 function v8LogToTicks (isolateLogPath, node) {
```

Comment:
let's rename this `node` parameter to something like `options`

Reply:
Indeed. Easier to understand.